samples/Quickstart/src/QuickstartFunction.cs

```diff
@@ -17,11 +17,14 @@ public QuickstartFunction(IPipeline<HttpRequestData, HttpResponseData> pipeline,
             _logger = loggerFactory.CreateLogger<QuickstartFunction>();
         }
 
-        [Function("QuickstartFunction")]
-        public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "Patient")] HttpRequestData req)
+        [Function("Patient")]
+        public async Task<HttpResponseData> PatientGet([HttpTrigger(AuthorizationLevel.Function, "get", "post", "put", "delete", Route = "Patient/{id?}")] HttpRequestData req)
         {
-            _logger.LogInformation("Quickstart pipeline started...");
-            return await pipeline.ExecuteAsync(req);
+            // This is what hooks up the Azure Function to the Custom Operation pipeline
+            _logger.LogInformation("Patient sample pipeline started...");
+            var responseData = await pipeline.ExecuteAsync(req);
+            responseData.Headers.Remove("Content-Length");
```

Comment:
Removing the Content-Length header is not a good practice.  The Content-Length would be set for the pipeline and would represent the length of the ContentString in the OperationContext. If this is faulting, then we need to debug it.

Reply:
Even if we are not adding extra content in the body, we still get the content length mismatch error. 
If you want to test, you can check the QuckStartSample_Post Test case in the QuickstartSample.Tests solution where postContent length and response content length are different.
We are still analyzing  it further and will update you.

Reply:
We have resolved this issue by changing some code in the AzureFunctionExtensions.cs file in the Pipeline Folder. We are reading the content length from Content.Headers. rather than content length. Please verify whether this implementation is correct or not.

![image](https://user-images.githubusercontent.com/109289871/187153682-9647672d-682e-4c9b-8fc0-c15b8b189ed4.png)